CHANGELOG.md

```diff
@@ -9,20 +9,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-### Added 
+### Added
+
+- Add option parameter `force` in list devices functions to allow force update `deviceInfoCache`.
```

Comment:
This should probably be more explanatory:

```
- Add a `force` parameter to use when listing devices. In some cases, builders
  need to delay the triggering of permission dialogs, _e.g._, when joining a
  meeting in view-only mode, and then later be able to trigger a permission
  prompt in order to show device labels. This parameter allows cached device
  labels to be forcibly discarded and recomputed after the device label trigger
  is run.
```

Reply:
Good explanation! Will replace the current one.

Reply:
Replaced.